templates/test-sample.php

```diff
@@ -2,7 +2,7 @@
 /**
```

Comment:
Can we rename this file to `test-sample.mustache`, just to make sure it's clear that it's a template now?